City-of-Helsinki/berth-reservations-ui
# VEN-892 | Reorganize municipalities list

## Description :sparkles:

This ranks Helsinki, Espoo, Vantaa and Kauniainen at the top of
the municipality select.

### Closes :no_good_woman:

[VEN-892](https://helsinkisolutionoffice.atlassian.net/browse/VEN-892)

## Testing :alembic:

### Manual testing :construction_worker_man:

- Fill any application to the point where address information should be provided
- The first 4 items of the municipality select should be Helsinki, Vantaa, Espoo and Kauniainen
